Microsoft.Toolkit.Uwp/IncrementalLoadingCollection/IncrementalLoadingCollection.cs

```diff
@@ -159,6 +160,18 @@ public IncrementalLoadingCollection(int itemsPerPage = 20, Action onStartLoading
         /// <seealso cref="IIncrementalSource{TSource}"/>
         public IncrementalLoadingCollection(TSource source, int itemsPerPage = 20, Action onStartLoading = null, Action onEndLoading = null, Action<Exception> onError = null)
         {
+            if (EqualityComparer<TSource>.Default.Equals(source, default(TSource)))
+            {
+                var type = typeof(TSource);
+                ConstructorInfo constructor = type.GetConstructor(new Type[0]);
+                if (constructor == null)
+                {
+                    throw new InvalidOperationException("TSource must have a parameterless constructor");
+                }
+
+                source = (TSource)constructor.Invoke(null);
+            }
+
```

Comment:
@deltakosh I tested it so TSource had a default constructor and it was instantiated and used.. then I added a param to constructor and ensure that it failed correctly and I then commited the code